cfme/automate/explorer/domain.py

```diff
@@ -292,13 +286,6 @@ def update(self, updates):
         view = self.create_view(DomainDetailsView, override=updates)
         assert view.is_displayed
         view.flash.assert_no_error()
-        if changed:
-            text = (updates.get('description', self.description)
-                    or updates.get('name', self.name))
-            view.flash.assert_message('Automate Domain "{}" was saved'.format(text))
```

Comment:
**Required** agree with removing this in this module but I think there should be a test checking these messages flash messages.

Reply:
@jawatts 
I agree. I am thinking of updating testcase ```test_domain_crud``` to check domain delete flash message.